composer.json

```
@@ -24,12 +24,12 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": "^7.2.5 || <8.2.0",
+        "php": "^7.2.5 || <9.0",
```

Maybe:

```suggestion
        "php": "^7.2.5 || ^8.0",
```

to keep it with `^` and `~` annotations?

---

I think it should be `"php": "^7.2.5 || ^8.0"` .